packages/template-cli/templates/src/ssr/package.json

```diff
@@ -1,5 +1,6 @@
 {
-  "name": "webapp",
+  "name": "PROJECT_NAME",
+  "version": "0.1.0",
```

Comment:
This shouldn't have a version.

Reply:
Ah I see, we log in app insights with the version. Do we need to? Else, default to 0.0.0?

Reply:
Yes. This is for app insights annotation.  What do we need to ? 

CSR is also versioned same way. So we are making it consistent